de.tu_bs.cs.isf.e4cf.parts.project_explorer/src/de/tu_bs/cs/isf/e4cf/parts/project_explorer/wizards/base_wizard/Wizard.java

```diff
@@ -0,0 +1,85 @@
+package de.tu_bs.cs.isf.e4cf.parts.project_explorer.wizards.base_wizard;

```

Comment:
This is a class that is never referenced or used across the whole framework...
I would suggest removing dead code but @KamilRosiak has the last word about it, as he requested a base wizard.